grabbit/build.gradle

```diff
@@ -70,6 +59,7 @@ jar.manifest {
     instruction 'Import-Package', 'org.springframework.aop.scope; version="[3.1,4.0)"'
     instruction 'Import-Package', 'org.springframework.scheduling.concurrent; version="[3.1,4.0)"'
     instruction 'Import-Package', "org.aopalliance.aop;version=1.0.0"
+    instruction 'Import-Package', "sun.misc"
```

Comment:
It's a required runtime dependency. It's optional for Groovy because it may or may not be called: Grabbit **_will_** be calling it.